baggers.js

```diff
@@ -13616,6 +13616,48 @@ var data = {
      "contacts": {
        "mail": "mfoucher@lucca.fr"
      }
+    },
+    {
+      "since": "2016-09-29",
+      "name": "Florian Verger",
+      "bio": "Ingénieur R&D chez Lucca, éditeur de logiciels SaaS de gestion RH.",
+      "picture": "https://secure.gravatar.com/avatar/a896cbaf08bb15196c309e8055e0b814?size=200",
+      "websites": [
+        {
+          "name": "lucca",
+          "url": "https://www.lucca.fr/"
+        },
+        {
+          "name": "LinkedIn",
+          "url": "https://www.linkedin.com/in/florian-verger-156a8a67"
+        }
+      ],
+      "location": "Nantes",
+      "sessions": [
+        {
+          "title": "Réécrivez l'histoire avec git",
+          "abstract": "Présentation sur des cas concrets afin de garder un historique git propre et lisible. Fini le commit 'fix de fix'",
+          "tags": [
+            "git",
```

Comment:
Just an opinion here but `git` as a tag should be enough. Plus `sourcetree`. Feel free to ignore my comment though.